src/webgpu/api/validation/encoding/programmable/pipeline_bind_group_compat.spec.ts

```diff
@@ -88,7 +91,36 @@ class F extends ValidationTest {
 
 export const g = makeTestGroup(F);
 
-g.test('it_is_invalid_to_draw_in_a_render_pass_with_missing_bind_groups')
+g.test('set_pipeline_or_bind_group_missing_in_render_pass')
+  .desc('Tests the bind groups required by the pipeline layout are required.')
+  .paramsSubcasesOnly(u =>
+    u
+      .combine('cmd', kDrawCmds)
+      .combine('setPipeline', [true, false] as const)
+      .combine('setBindGroup', [true, false] as const)
+  )
+  .unimplemented();
+
+g.test('set_pipeline_or_bind_group_missing_in_compute_pass')
+  .desc('Tests the bind groups required by the pipeline layout are required.')
+  .paramsSubcasesOnly(u =>
+    u
+      .combine('cmd', kDispatchCmds)
+      .combine('setPipeline', [true, false] as const)
+      .combine('setBindGroup', [true, false] as const)
+  )
+  .unimplemented();
+
+g.test('bind_groups_mismatch_with_pipeline_layout_in_render_pass')
+  .desc(
+    `
+  Tests the bind groups unused by the pipeline layout can be set or not.
+  TODO:
+  Set 3 bindGroups
+  - all 2 needed bindGroups are set
```

Comment:
nit: this TODO is unclear, I assume it's supposed to be the "(Even if e.g. bind groups 0 and 2 are used, but 1 is unused.)" from above?

Reply:
I think this might be, with 3 bindGroups, we could have a new case:
- 2 bind group are required but we set 1 required bind group and 2 unused ones. -- should fail